pkgs/development/libraries/libff/default.nix

```diff
@@ -14,6 +14,10 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [ "-DWITH_PROCPS=Off" ];
 
+  postPatch = stdenv.lib.optionalString stdenv.isDarwin ''
```

Comment:
do you mind adding a comment, so there's some context in the future

Reply:
done